lib/ruby_warrior/note.rb

```diff
@@ -2,9 +2,12 @@
 
 module RubyWarrior
   class Note < ActiveRecord::Base
-    establish_connection adapter: "sqlite3", database: "warriors/foobar.db"
-    connection.create_table table_name, if_not_exists: true do |t|
-      t.string :body
+    establish_connection adapter: "sqlite3", database: "warriors/map_survey.db"
+    connection.create_table :notes, if_not_exists: true do |t|
+      t.column :level_number, :int, limit: 3, :null => false
```

Comment:
This is actually difficult to do. I was looking at it yesterday, and the `Player` class currently doesn't have a way of accessing the current level number; we would need to pipe it in somehow